db/internal_stats.h

```diff
@@ -168,6 +168,12 @@ class InternalStats {
       this->num_output_files += stats.num_output_files;
       this->num_output_files_blob += stats.num_output_files_blob;
     }
+
+    bool IsEmpty() const {
```

Comment:
not used?

Reply:
Oh yeah, this was going to be used, then not. ;)